hoot-cmd/src/main/cpp/hoot/cmd/HelpCmd.cpp

```diff
@@ -22,7 +22,7 @@
  * This will properly maintain the copyright information. Maxar
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015, 2016, 2017, 2018, 2019, 2020, 2021 Maxar (http://www.maxar.com/)
+ * @copyright Copyright (C) 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022 Maxar (http://www.maxar.com/)
```

Comment:
@bmarchant Thank you for the information.  I was unaware that copyrights only update when a file is modified.  Would you like me to revert the extraneous copyright year updates?

Reply:
I might have to just close this PR and open a new one.  Since, if I revert the copyright changes, the file is still modified and the copyright will update.

Reply:
Yeah, probably the best way to do it would be to copy the 9 or so files that you did update into a new branch, commit that branch, create a new PR, and close this one.

Reply:
@bmarchant FYSA - new PR here: https://github.com/ngageoint/hootenanny/pull/5409